### PR TITLE
Fix invalid config key msg for dashbord

### PR DESCRIPTION
**Pull request by mniewrzal**
Fix or workaround (not sure) for:
image

### CODE REVIEW

File: pkg/process/exec_conf.go [hunk 1]
```diff
@@ -179,19 +179,21 @@ func cleanup(cmd *cobra.Command) {
...
-			if cmd.Flags().Lookup(key) == nil {
-				// flag couldn't be found
-				brokenKeys = append(brokenKeys, key)
-			} else {
-				oldChanged := cmd.Flag(key).Changed
-				err := cmd.Flags().Set(key, vip.GetString(key))
-				if err != nil {
-					// flag couldn't be set
-					brokenVals = append(brokenVals, key)
+		if cmd.Annotations["type"] != "helper" {
```
Comment (egonelbre) [inline pkg/process/exec_conf.go]: Let's still keep the check for broken values. You can just skip the printing below, we can still collect the broken values.

Comment (mniewrzal) [reply pkg/process/exec_conf.go]: ok

File: pkg/process/exec_conf.go [hunk 2]
```diff
@@ -214,12 +214,14 @@ func cleanup(cmd *cobra.Command) {
...
-			logger.Sugar().Infof("Invalid configuration file key: %s", key)
-		}
-		for _, key := range brokenVals {
-			logger.Sugar().Infof("Invalid configuration file value for key: %s", key)
+		if cmd.Annotations["type"] != "helper" {
+			// okay now that logging is working, inform about the broken keys
+			for _, key := range brokenKeys {
+				logger.Sugar().Infof("Invalid configuration file key: %s", key)
+			}
+			for _, key := range brokenVals {
```
Comment (egonelbre) [inline pkg/process/exec_conf.go]: I think we still want to print invalid values?

Comment (mniewrzal) [reply pkg/process/exec_conf.go]: yes, you are right

**Pull request merged**